orders/migrations/0003_static_orders.py

```diff
@@ -0,0 +1,79 @@
+from django.db import migrations, transaction
+from django.utils import timezone
+
+DOG_OWNER_ID = ['4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21',
+                '22', '23', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19',
+                '20', '21', '22', '23']
+
+DAYCARE_ID = ['4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21',
+              '22', '23', '23', '22', '21', '20', '19', '18', '17', '16', '15', '14', '13', '12', '11', '10', '9', '8',
+              '7', '6', '5', '4']
+
+START_DATE = [timezone.datetime(2022, 3, 23), timezone.datetime(2022, 5, 23), timezone.datetime(2022, 2, 23),
+              timezone.datetime(2022, 6, 7), timezone.datetime(2022, 7, 15), timezone.datetime(2021, 1, 20),
+              timezone.datetime(2023, 6, 6), timezone.datetime(2021, 5, 6), timezone.datetime(2023, 4, 20),
+              timezone.datetime(2022, 3, 30), timezone.datetime(2022, 7, 25), timezone.datetime(2022, 10, 10),
+              timezone.datetime(2022, 12, 15), timezone.datetime(2022, 5, 11), timezone.datetime(2022, 9, 10),
+              timezone.datetime(2022, 6, 11), timezone.datetime(2022, 5, 5), timezone.datetime(2022, 2, 22),
+              timezone.datetime(2022, 7, 8), timezone.datetime(2022, 8, 1), timezone.datetime(2022, 1, 1),
+              timezone.datetime(2022, 7, 7), timezone.datetime(2022, 7, 20), timezone.datetime(2022, 8, 27),
+              timezone.datetime(2022, 1, 15), timezone.datetime(2022, 12, 18), timezone.datetime(2022, 9, 23),
+              timezone.datetime(2022, 11, 12), timezone.datetime(2022, 11, 11), timezone.datetime(2022, 1, 23),
+              timezone.datetime(2022, 6, 12), timezone.datetime(2022, 2, 15), timezone.datetime(2022, 3, 11),
+              timezone.datetime(2022, 1, 17), timezone.datetime(2022, 12, 23), timezone.datetime(2022, 3, 12),
+              timezone.datetime(2022, 3, 5), timezone.datetime(2022, 4, 4), timezone.datetime(2022, 5, 15),
+              timezone.datetime(2022, 5, 5)]
+
+END_DATE = [timezone.datetime(2022, 3, 30), timezone.datetime(2022, 5, 25), timezone.datetime(2022, 3, 1),
```

Comment:
maybe instead of keeping the END_DATE list, use START_DATE + random number of days (in our order days range)

Reply:
It can be done, but what's the benefit of using it?

Reply:
maybe I'm wrong, but for me, it looks more elegant than the (not so pretty) END_DATE list